se2_planning/src/GridMapLazyStateValidator.cpp

```diff
@@ -129,6 +168,121 @@ bool isInCollision(const SE2state& state, const std::vector<Vertex>& footprint,
   return false;
 }
 
+// TODO Not working with updated state space bounds
```

Comment:
what fails here? This approach that is commented out is significantly faster as opposed to using grid_map iterators. We should try to keep it!

Reply:
With the current implementation of isInCollision and isTraversable, I get the strange behavior that along the x-axis the state validator accepts all possible positions, along the y-axis only the ones that lie in the map.